core/src/debug_client.rs

```diff
@@ -0,0 +1,65 @@
+use reqwest;
+use prost::Message;
+use temporal_sdk_core_protos::temporal::api::history::v1::History;
+use std::time::Duration;
+
+struct DebugClient {
+    debugger_url: String,
+    _client: reqwest::Client
+}
+
+impl DebugClient {
+
+    fn new(url: String) -> DebugClient {
+        DebugClient {
+            debugger_url: url,
+            _client: reqwest::Client::new(),
+        }
+    }
+
+    async fn get_history(&self) -> Result<History, anyhow::Error> {
+        let url = self.debugger_url.clone() + "/history";
+        let resp = self._client.get(url)
+            .header("Temporal-Client-Name", "temporal-core")
+            .header("Temporal-Client-Version", "0.1.0")
+            .send()
+            .await?;
+
+        let bytes = resp.bytes().await?;
+        Ok(History::decode(bytes)?) // decode_length_delimited() does not work
+
+    }
+
+    async fn post_wft_started(&self, event_id: &i64) -> bool {
+        let url = self.debugger_url.clone() + "/current-wft-started";
+        let resp = self._client.get(url)
+            .header("Temporal-Client-Name", "temporal-debug-core")
```

Comment:
It doesn't tremendously matter what these are but they should at least be the same for these two methods. You can lift both of these up into constants.